src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
+        storedTasks[unmarkIndex].setDone(false);
+        System.out.println("OK, I've marked this task as not done yet.");
+        System.out.println(storedTasks[unmarkIndex].getTypeIcon() +
+                storedTasks[unmarkIndex].getStatusIcon() + " " + storedTasks[unmarkIndex].getDescription());
+    }
+
+    private static void markTask(Task[] storedTasks, String[] commandLine) {
+        int taskIndex = Integer.parseInt(commandLine[1]) - 1;
+        storedTasks[taskIndex].setDone(true);
+        System.out.println("Nice! I've marked this task as done.");
+        System.out.println(storedTasks[taskIndex].getTypeIcon() +
+                storedTasks[taskIndex].getStatusIcon() + " " + storedTasks[taskIndex].getDescription());
+    }
+
+    private static void addEventTask(Task[] storedTasks, String[] commandLine) {
+        String[] eventString = commandLine[1].split("/from");
+        String[] eventStartEnd = eventString[1].split("/to");
+        Event ev = new Event(eventString[0], eventStartEnd[0], eventStartEnd[1]);
+        storedTasks[taskNum] = ev;
+        addTaskMessage();
+        System.out.println("  " + ev.getTypeIcon() +
+                ev.getStatusIcon() + " " + ev.getDescription() + "(from: " + ev.getStart() + " to: " + ev.getEnd() +")");
+        taskNum++;
+        displayTasksNum();
+    }
+
+    private static void addTodoTask(Task[] storedTasks, String[] commandLine) {
+        Todo td = new Todo(commandLine[1]);
+        storedTasks[taskNum] = td;
+        addTaskMessage();
+        System.out.println("  " + td.getTypeIcon() + td.getStatusIcon() + " " + td.getDescription());
+        taskNum++;
+        displayTasksNum();
+    }
+
+    private static void addDeadlineTask(Task[] storedTasks, String[] commandLine) {
+        String[] deadlineString = commandLine[1].split("/by");
+        Deadline dl = new Deadline(deadlineString[0], deadlineString[1]);
+        storedTasks[taskNum] = dl;
+        addTaskMessage();
+        System.out.println("  " + storedTasks[taskNum].getTypeIcon() +
+                dl.getStatusIcon() + " " + dl.getDescription() + "(by: " + dl.getBy() + ")");
+        taskNum++;
+        displayTasksNum();
+    }
+
+    private static void showGreetings() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+    private static void printLineBreak() {
+        System.out.println("--------------------------------------------------------------");
+    }
+
+    private static void addTaskMessage() {
+        System.out.println("Got it. I've added this task.");
+    }
+    private static void listMessage() {
```

Comment:
Perhaps you should use the name to explain the function.
```suggestion
    private static void printListMessage() {
```

Reply:
thanks, will do